angular_components/1-base/lib/src/settings/settings.dart

```diff
@@ -2,8 +2,8 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-import 'package:angular2/angular2.dart';
-import 'package:components_codelab/src/lottery/lottery.dart';
+import 'package:angular/angular.dart';
+import '../lottery/lottery.dart';
```

Comment:
I thought we preferred same-package imports to `../` imports (although here `../` seems cleaner). @matanlurey or @munificent, do we have a recommended style?